CanvasColorSpaceProposal.md

```diff
@@ -43,21 +43,18 @@ IDL:
 <pre>
 enum CanvasColorSpace {
   "srgb", // default
-  "rec2020",
-  "p3",
+  "extended-srgb",
```

Comment:
Do we really need an explicit extended-srgb mode? We could just define the behavior of the  "srgb" + "float16" combo to be what you defined as extended-srgb. Otherwise what is the difference between the two? Clamping?  I would rather avoid a mode where all intermediate values need to be clamped. Would burden to implement and support that mode, for limited usefulness IMHO.

Reply:
That sounds good. Removed.